neoutils/connection.go

```diff
@@ -37,6 +41,18 @@ func Connect(neoURL string, conf *ConnectionConfig) (NeoConnection, error) {
 		conf = DefaultConnectionConfig()
 	}
 
+	if !conf.BackgroundConnect {
+		return connectDefault(neoURL, conf)
+	} else {
+		f := func() (NeoConnection, error) {
+			return connectDefault(neoURL, conf)
+		}
+		return connectAuto(neoURL, f, 30*time.Second)
```

Comment:
Is the 30 seconds a first guess, or a calculated amount of time to wait between connection attempts?